### ISSUE TITLE
[Adaptive Cards - Input]: While navigating to menu items under 'What color do you want'? combo box, selected menu items are not visually appeared as selected. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#103]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**
1. Launch the Adaptive cards app on Android and turn on the Talkback.
2. Navigate to 'Import' control and select it.
3. Navigate to 'Sample cards' and select it.
4. Navigate to available dropdown under 'Sample cards' and select it.
5. Navigate all the controls under dropdown and select 'Stock Update'.
6. Navigate to 'Visual' tab control and select it.
7. Navigate to 'Input' option controls under 'Visual'.
8. Navigate the elements present under 'Inputs' and observe the issue.

**Actual Result:**
While navigating to the selected menu items present under  'What color do you want'? combo box, selected menu item is not visually appearing as selected as there is no tick box is present.

**Expected Result:**
While navigating to the selected menu items present under  'What color do you want'? combo box, selected menu item should be visually appeared as selected, There should be tick mark present for the selected items.

**User Impact:**
It will impact the visually impaired user as they will not able to know which option is selected under  'What color do you want'? combo box.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/8fd05ecc-cc13-4ed5-9db4-117a8c855674

![While navigating to menu items under 'What color do you want' combo box, selected menu items are not visually appeared as selected](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/77e0ce74-f5b6-477d-9c6c-73b06e9510b4)

### Card JSON

```JSON
NA
```
